src/App.scss

```diff
@@ -4,8 +4,8 @@
   position: relative;
   overflow-x: hidden;
   width: 100%;
-  height: 100vh;
+  height: 100%;
```

Comment:
Hey, what's the reason behind changing the app container height unit from `vh` to `%`? Because I see below that you changed the height for page containers e.g. from `calc(100% - 140px);` to `calc(100vh - 140px);` so the opposite way.

Reply:
`vh` was breaking fullpage.js on Safari, this was the reason behind the scroll bug.

Reply:
Right, thank you for the clarification! :+1:  I found [the same issue](https://github.com/alvarotrigo/fullPage.js/issues/3396) on fullPage.js Github